src/gr_poly/contains_zero_poly.c

```diff
@@ -0,0 +1,23 @@
+/*
+    Copyright (C) 2024 Ricardo Buring
+
+    This file is part of FLINT.
+
+    FLINT is free software: you can redistribute it and/or modify it under
+    the terms of the GNU Lesser General Public License (LGPL) as published
+    by the Free Software Foundation; either version 3 of the License, or
+    (at your option) any later version.  See <https://www.gnu.org/licenses/>.
+*/
+
+#include "gr_vec.h"
+#include "gr_poly.h"
+
+/* todo: faster code when possible */
```

Comment:
What do you mean by this comment? Isn't this the fastest general way to do it for polynomials?

Reply:
Sorry, that was copied from `is_zero`. I can remove it when there is agreement on how to name the functions.